src/main/java/io/supertokens/webserver/api/jwt/JWTSigningAPI.java

```diff
@@ -68,7 +69,7 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws I
                     new WebserverAPI.BadRequestException("validity must be greater than or equal to 0"));
         }
 
-        boolean useDynamicKey = version.equals("2.18") &&
+        boolean useDynamicKey = version.equals(SemVer.v2_19) &&
                 Boolean.FALSE.equals(InputParser.parseBooleanOrThrowError(input, "useStaticSigningKey", true));
```

Comment:
useStaticSigningKey should be non-nullable? So the last argument should be false? If it's true, it means that the `parseBooleanOrThrowError` will return `null`, which means the equals will return false, and we will not use dynamic keys.

If we want to keep it nullable, then we should make it more clear -> first read it from the input, if it's null, set the value of useStaticSigningKey to true, and then use the equals function. Easier to read is always better